app/App.tsx

```diff
@@ -1,9 +1,5 @@
 import {
-  Stacks,
-  Screens,
   AgentProvider,
-  AnimatedComponentsProvider,
-  animatedComponents,
```

Comment:
Why were these two removed? I still see `useAnimatedComponents` throughout Bifold

Reply:
not sure, must have gotten deleted when I was moving the settings to container-imp